.github/workflows/autolabel_action.yaml

```diff
@@ -0,0 +1,39 @@
+name: Apply at most one `t-`label to PRs
+
+on: pull_request
+
+jobs:
+  apply_one_t_label:
+    runs-on: ubuntu-latest
+    # Don't run on forks, where we wouldn't have permissions to add the label anyway.
+    if: github.repository == 'leanprover-community/mathlib4'
+    permissions:
+      issues: write
+      checks: write
+      pull-requests: write
+      contents: read
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - run: |
+          ./scripts/autolabels.sh long
+          printf 'Just the labels: "%s"\n' "$(./scripts/autolabels.sh)"
+          LABELS="$(./scripts/autolabels.sh)"
+          printf $'gh issue edit \'%s\' --add-label \'%s\'\n' "$NUMBER" "$LABELS"
```

Comment:
How about moving this to the branch actually adding the labels, such as printing
"About to add labels by calling `'gh issue edit \'%s\' --add-label \'%s\'\n' "$NUMBER" "$LABELS"`"

Reply:
I need to experiment with this: `$NUMBER` is actually read off via the workflow API and I do not really know if I can use it in a script that the API calls.

Besides, I am planning to rewrite the whole script in Lean, since I think that it will be easier to understand and maintain.

Reply:
Waking up, I was wondering how long rewriting this in Python or Lean would take.
A Lean rewrite would be perfect!